logging/CHANGELOG.md

```diff
@@ -4,6 +4,29 @@
 
 [1]: https://pypi.org/project/google-cloud-logging/#history
 
+## 1.4.1
+
+### Implementation Changes
+  - Logging pass in grace period and batch size to workers (#4369)
+  - Unique Writer Identity (#4595, #4704, #4706, #4708)
+  - Making Sink.filter Optional (#4699)
+  - Capture Server-generated writerIdentity during LogSink.create (#4707)
+  - Generate GAPIC and Remove GAX (#4759)
+  - Move code path of get_gae_labels() to emit() (#4824)
+
+### Dependencies
+  - Pinning `Django` test dependency to < 2.0 in Python 2.7 (#4519)
+
+### Tests
+  - Test for retrieve_metadata_server against transparent DNS proxies (#4698)
+  - Adding cleanup for Pub/Sub topic in logging system test (#4532)
+
+### Minor Bug Fixes
```

Comment:
Either these should have a one-sentence summary of what they fix or they should be omitted.